app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
```

Comment:
chị có suggest là: cem nên đưa code find user ra 1 method riêng nhé
nhớ xử lý user not found luôn 

còn ở đây chỉ check `!user.activated? && user.authenticated?(:activation, params[:id])`